src/librustc_typeck/astconv.rs

```diff
@@ -12,32 +11,34 @@ use crate::lint;
 use crate::middle::lang_items::SizedTraitLangItem;
 use crate::middle::resolve_lifetime as rl;
 use crate::namespace::Namespace;
+use crate::require_c_abi_if_c_variadic;
+use crate::util::common::ErrorReported;
+use crate::util::nodemap::FxHashMap;
+
 use rustc::lint::builtin::AMBIGUOUS_ASSOCIATED_ITEMS;
 use rustc::traits;
 use rustc::ty::{self, DefIdTree, Ty, TyCtxt, Const, ToPredicate, TypeFoldable};
 use rustc::ty::{GenericParamDef, GenericParamDefKind};
 use rustc::ty::subst::{self, Subst, InternalSubsts, SubstsRef};
 use rustc::ty::wf::object_region_bounds;
 use rustc_target::spec::abi;
-use crate::require_c_abi_if_c_variadic;
-use smallvec::SmallVec;
+
 use syntax::ast;
 use syntax::errors::pluralize;
 use syntax::feature_gate::{GateIssue, emit_feature_err};
 use syntax::util::lev_distance::find_best_match_for_name;
 use syntax::symbol::sym;
 use syntax_pos::{DUMMY_SP, Span, MultiSpan};
-use crate::util::common::ErrorReported;
-use crate::util::nodemap::FxHashMap;
+
+use errors::{Applicability, DiagnosticId};
+use rustc_data_structures::fx::FxHashSet;
+use rustc_error_codes::*;
+use smallvec::SmallVec;
 
 use std::collections::BTreeSet;
 use std::iter;
 use std::slice;
 
-use rustc_data_structures::fx::FxHashSet;
-
-use rustc_error_codes::*;
-
 #[derive(Debug)]
 pub struct PathSeg(pub DefId, pub usize);
```

Comment:
Please do not include unrelated changes, it pointlessly makes review hard (if you use `git gui` you can selectively commit some changes and discard the rest, for example).

Reply:
Well at least I'm changing this file... seems like too minor a quibble here.